Modules/GLO/src/ITSTPCMatchingTask.cxx

```diff
@@ -130,6 +83,7 @@ void ITSTPCMatchingTask::startOfActivity(const Activity& activity)
 {
   ILOG(Debug, Devel) << "startOfActivity " << activity.mId << ENDM;
   mMatchITSTPCQC.reset();
+  mIsPbPb = activity.mBeamType == "Pb-Pb";
```

Comment:
```suggestion
  mIsPbPb = activity.mBeamType == "PbPb";
```
Use `PbPb`, as we will move from LHC-provided beam type to the one provided by `pdp_beam_type`. See https://github.com/AliceO2Group/QualityControl/pull/2540

Reply:
thanks!
N.B. an enum or some getter `isPbPb` provided by the activity would be nicer than a string comp.

Reply:
would decouple this for users as-well

Reply:
I agree.